### PR TITLE
error handling

### PR DESCRIPTION
**Pull request by charmingduchess**
rearrange things and fix error handling so 400 errors are returned as such!

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -16,17 +16,19 @@
 def handler(event, context):
     load_env_file(os.environ['ENVIRONMENT'], 'config/{}.yaml')
     init()
```
[the remark this answers is not in the document]
Comment (charmingduchess) [reply main.py]: updated!

File: main.py [hunk 2]
```diff
@@ -16,17 +16,19 @@
...
     method = event.get('httpMethod')
-    params = event.get('queryStringParameters')
-    (location_codes, fields) = parse_params(params)
     if method != 'GET':
         return create_response(501, 'LocationsService only implements GET \
-endpoints')
+            endpoints')
+    path = event.get('path')
     if path == '/docs/locations':
         load_swagger_docs()
```
[the remark this answers is not in the document]
Comment (charmingduchess) [reply main.py]: returned!

File: lib/location_lookup.py [hunk 3]
```diff
@@ -20,9 +25,11 @@ def init():
             raise MissingEnvVar('S3_LOCATIONS_FILE')
         s3_client = S3Client(bucket, resource)
         CACHE['s3_locations'] = s3_client.fetch_cache()
+        CACHE['s3_locations']['updatedAt'] = time.time()
```
Comment (nonword) [inline lib/location_lookup.py:28]: `updated_at` or `updatedAt`?

**Review by nonword — APPROVED** [review]
Looks good

**Pull request merged**